src/wrapper/objects/auto_byte_array.rs

```diff
@@ -73,6 +73,11 @@ impl<'a, 'b> AutoByteArray<'a, 'b> {
     pub fn is_copy(&self) -> bool {
         self.is_copy
     }
+	
+	/// Returns the size of this byte array as a jsize
+	pub fn len(&self) -> jsize {
+		self.env.get_array_length(*self.obj).expect("Couldn't fetch byte array length")
+	}
```

Comment:
I don't think this method is a good idea, as it's relatively inefficient, and it's for obtaining data that you probably have already

Better pass the `len` as a parameter in the main (`define_class_bytearr`) method.